libs/ui-watt/src/lib/components/clipboard/watt-copy-to-clipboard.directive.spec.ts

```diff
@@ -17,13 +17,13 @@
 import { render, screen } from '@testing-library/angular';
 import userEvent from '@testing-library/user-event';
 
-import { WattToastModule } from '../toast';
+import { WattToastComponent } from '../toast';
 import { WattCopyToClipboardDirective } from './watt-copy-to-clipboard.directive';
 
 describe(WattCopyToClipboardDirective.name, () => {
   it('shows toast on click', async () => {
     await render(`<span wattCopyToClipboard>Text</span>`, {
-      imports: [WattToastModule.forRoot(), WattCopyToClipboardDirective],
```

Comment:
suggestion: a bit weird that this works. Probably it's because `WattToastComponent` imports `MatLegacySnackBarModule`, which in turn provides the necessary providers. Instead of importing `WattToastComponent`, which looks unrelated to this directive anyway, let's use the `importProvidersFrom(MatLegacySnackBarModule)` pattern.